repos/fdbt-netex-output/src/netex-convertor/sharedHelpers.ts

```diff
@@ -95,20 +96,22 @@ export const getProfileRef = (
     };
 };
 
-export const getUserProfile = (user: User | GroupCompanion, index: number): NetexObject => ({
-    version: '1.0',
-    id: `op:${user.passengerType}-${index}`,
-    Name: { $t: user.passengerType },
-    TypeOfConcessionRef: {
-        version: 'fxc:v1.0',
-        ref: `fxc:${
-            user.passengerType === 'anyone' || user.passengerType === 'adult' ? 'none' : snakeCase(user.passengerType)
-        }`,
-    },
-    MinimumAge: { $t: user.ageRangeMin || null },
-    MaximumAge: { $t: user.ageRangeMax || null },
-    ProofRequired: { $t: user.proofDocuments?.join(' ') || null },
-});
+export const getUserProfile = (user: User | GroupCompanion, index: number): NetexObject => {
+    const passengerType = isPassengerType(user.passengerType) ? user.passengerType : '';
```

Comment:
I think we need to be throwing an error here if the passenger type is not one of the accepted ones, because something has gone very very wrong if so, and we need to be alerted to it.

Reply:
so something like 

`if (!isPassengerType(user.passengerType)) { throw new Error('Passenger type received was not one of the accepted enumerations.')`

Reply:
Alternatively you could move all of this into a function, and have that function be the one to throw the error, to make this function a little bit more concise. If you do this, which I think makes sense, we could execute the function at the start of the netex generator, not all the way down here.

Reply:
[476457a](https://github.com/fares-data-build-tool/create-data/pull/564/commits/476457a1a23051d392667a04692cd604b61afa5b)